python/cuspatial/cuspatial/io/shapefile.py

```diff
@@ -8,7 +8,28 @@
 
 
 def read_polygon_shapefile(filename):
-    """Reads a shapefile into GPU memory."""
+    """
+    Reads an ESRI shapefile into GPU memory.
+
+    Parameters
+    ----------
+    filename : str, pathlike
+        ESRI Shapefile file path (usually ends in ``.shp``)
+
+    Returns
+    -------
+    result  : tuple (cudf.Series, cudf.Series, cudf.DataFrame)
+    poly_offsets   : cudf.Series(dtype=np.int32)
+        Offsets of the first ring in each polygon
+    ring_offsets   : cudf.Series(dtype=np.int32)
+        Offsets of the first point in each ring
+    points  : cudf.DataFrame
```

Comment:
It's strange that we don't return a single DataFrame, or four Series... But this is a documentation PR. :)

Reply:
I believe all these could be different lengths?

Reply:
Right. That makes sense. 🤦